source/backend/api/Areas/Acquisition/Controllers/InterestHolderController.cs

```diff
@@ -55,9 +55,9 @@ public InterestHolderController(IAcquisitionFileService acquisitionService, IMap
         [ProducesResponseType(typeof(IEnumerable<InterestHolderModel>), 200)]
         [SwaggerOperation(Tags = new[] { "acquisitionfile" })]
         [TypeFilter(typeof(NullJsonResultFilter))]
-        public IActionResult GetAcquisitionFileInterestHolders([FromRoute] long id)
+        public IActionResult GetAcquisitionFileInterestHolders([FromRoute] long id, bool? fullDetails)
         {
-            var interestHolders = _acquisitionService.GetInterestHolders(id);
+            var interestHolders = _acquisitionService.GetInterestHolders(id, fullDetails ?? false);
```

Comment:
Yes, I thought of adding the extra joined tables to the original query but this would bring two more related tables into the query "interest holder" -> person -> address (new) -> country (new). 
This is only needed on the generation side as we want to display full addresses including country/province/etc but for the original stakeholder tabs where contacts are added/removed as interest holders on an Acquisition File, the addresses are not needed.

I will remove the extra logic and let's discuss a pattern moving forward when Devin comes back. We have talked in the past of having two ways of getting information (the lightweight model vs the fat model with many sub-entities loaded)